kobocat-template/templates/legacy_banner.html

```diff
@@ -0,0 +1,389 @@
+    color: hsl(229, 17%, 43%) !important;
+  }
+  
+  .legacy-banner__button.legacy-banner__button--gray:hover {
+    color: hsl(229, 17%, 33%) !important;
+  }
+
+  .legacy-banner-bar .legacy-banner__button {
+    float: right;
+    margin: 25px 0 25px 30px;
+  }
+  
+  .legacy-banner-popup__footer {
+    margin: 40px auto;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+  }
+  
+  .legacy-banner-popup__footer-left {
+    flex: 1;
+  }
+  
+  .legacy-banner-popup__footer * + * {
+    margin-left: 20px;
+  }
+
+  .legacy-banner-button__no_projects_override {
+    padding: 0 30px;
+    line-height: 40px;
+  }
+</style>
+
+<!-- 
+THE POPUP 
+-->
+<div
+  id="legacy-banner-popup-upgrade"
+  class="legacy-banner-popup legacy-banner-popup--hidden"
+>
+  <section class="legacy-banner-popup__content">
+    <section class="legacy-banner">
+      <h1 class="legacy-banner__header">
+        <svg
+          class="legacy-banner__icon"
+          viewbox="0 0 20 20"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            fill-rule="evenodd"
+            clip-rule="evenodd"
+            d="M11 11H9V6H11V11Z"
+            fill="#FF556A"
+          />
+          <path
+            fill-rule="evenodd"
+            clip-rule="evenodd"
+            d="M11 13C11 13.552 10.552 14 10 14C9.448 14 9 13.552 9 13C9 12.448 9.448 12 10 12C10.552 12 11 12.448 11 13Z"
+            fill="#FF556A"
+          />
+          <path
+            fill-rule="evenodd"
+            clip-rule="evenodd"
+            d="M10 2C5.589 2 2 5.589 2 10C2 14.411 5.589 18 10 18C14.411 18 18 14.411 18 10C18 5.589 14.411 2 10 2ZM10 20C4.486 20 0 15.515 0 10C0 4.486 4.486 0 10 0C15.514 0 20 4.486 20 10C20 15.515 15.514 20 10 20Z"
+            fill="#FF556A"
+          />
+        </svg>
+        <span>{% trans "Changes to the legacy interface on 14 June 2021" %}</span>
+      </h1>
+
+      <p class="legacy-banner__text">
+        {% trans "On 14 June 2021 we removed a number of components of this legacy user interface for viewing project data (also known as kobocat). Thank you to everyone who has participated in our survey over the last months and for responding <a href='https://community.kobotoolbox.org/t/contemplating-the-future-of-kobocat/2743' >to the forum discussion</a >. This updated version has allowed us to upgrade kobocat’s source code and dependencies to ensure proper data security in the long run, and lower the amount of work needed to maintain it. You can notice the following changes:"%}
+      </p>
+
+      <ul class="legacy-banner__text legacy-banner__text--list">
+        <li>
+          {% trans "Form uploads are only supported through the regular interface. Note that even if a setting form configuration is not supported yet in the formbuilder, you can deploy the form from there and it will work the same way as deployed through the legacy interface."%}
```

Comment:
This is a confusing sentence... I'm not sure I understand what's going on.
> Note that even if a setting form configuration is not supported yet in the formbuilder, you can deploy the form from there and it will work the same way as deployed through the legacy interface.

Reply:
@joshuaberetta I did not create this sentence myself. I just restored what was pushed to `master` to the legacy banner. 
You may want to discuss about it with a native English speaker.

Reply:
I see 👍 since it's already live on production I guess let's just leave it as is.